go-controller/pkg/kubevirt/dhcp.go

```diff
@@ -130,6 +131,7 @@ func composeDHCPOptions(controllerName, namespace, vmName string, dhcpOptions *n
 			libovsdbops.NamespaceKey:      namespace,
 		})
 	dhcpOptions.ExternalIDs = dhcpvOptionsDbObjectID.GetExternalIDs()
+	dhcpOptions.ExternalIDs[OvnZoneExternalIDKey] = OvnLocalZone
```

Comment:
This belongs to the third commit

Reply:
I prefer to group all the "OvnZoneExternalIDKey" changes together so it's easier to review.